include/gillwald/geometry.hpp

```diff
@@ -6,13 +6,19 @@
 #include <vector>
 
 namespace geometry {
+
+	struct Cell {
+	  int x, y;
+	};
+
   /**
    * @brief Creates discretized line between two end points
    * @param x0 X coordinate of starting point
    * @param y0 Y coordinate of starting point
    * @param x1 X coordinate of ending point
    * @param y1 Y coordinate of ending point
    * @returns Pixel coordinates representing line
-   */
+   */	
   std::vector<std::vector<int>> bresenham_conversion(int x0, int y0, int x1, int y1);
+  std::vector<Cell> raytraceLine_conversion(const Cell& start, const Cell& end);
```

Comment:
Can you add doxygen comments? Also, don't mix camel case and snake case. Let's stick to snake case.

Reply:
Done